storage/src/test/scala/coop/rchain/storage/StorageActionsTests.scala

```diff
@@ -39,6 +41,12 @@ abstract class StorageActionsTests extends FlatSpec with Matchers with OptionVal
     }
 
     r shouldBe None
+
+    store.withTxn(store.createTxnWrite()) { txn =>
+      store.removeA(txn, key.head, 0)
+    }
+
+    store.isNoGarbage shouldBe true
```

Comment:
I have mixed feelings about doing this here.  I recognize that it's a simple way to test this functionality without having to write separate tests, but I also think it's wrong to be modifying the store directly in a test like this where we are only using the public API of `produce` and `consume` to modify the store.  

Perhaps we should write separate unit tests to exercise this functionality.